### ISSUE TITLE
The usage of `--vpp-denoise-dct` with `--crop` or setting switches to copy HDR information results in erroneous video.

### ISSUE THREAD

**Issue by quamt** [rigaya/NVEnc#567]
When using `--crop` to remove black bars and `--vpp-denoise-dct` together with HDR info copying settings, the resulting video output is malformed.

There seem to be some issues with black bars, where they are either not cropped or cropped in the wrong position. For instance, only the bottom bar is removed while the top one remains. Additionally, there is a problem with the HDR information, which appears to be delayed or creates an afterimage effect.

When `--vpp-denoise-dct`  is switched off, it is all okay.

**Comment by rigaya** [comment]
Actually, --crop seems to be broken from NVEnc 7.42. I'll check and fix in the next release.

**Comment by quamt** [comment]
Ok. That's strange because it works fine with the latest version. When I tested it earlier.
Only when I use that with the combo I do get issues.

**Comment by rigaya** [comment]
[NVEnc 7.45](https://rigaya34589.blog.fc2.com/blog-entry-1796.html) should have fixed --crop issue, please have try.

**Comment by babiulep** [comment]
Of course N=1, but it seems to be working again! I've tried --vpp-deinterlace adaptive --crop 0,180,0,180 and  --vpp-deinterlace adaptive --crop 0,180,0,180 --vpp-denoise-dct --vpp-unsharp weight=0.75. Both combinations are working now.

P.S.: I'm compiling NVEnc myself (gcc 12.3.1) and use the latest ffmpeg libraries (v6.x) and Cuda 12.4. I got a compile error in NVEncCore/rgy_input_avcodec.cpp (r 2946) and changed m_Demux.video.frame->pkt_duration to m_Demux.video.frame->duration to get it working.
